packages/default-theme/pages/_.vue

```diff
@@ -49,8 +49,7 @@ export default {
       cmsPageName: name,
       page: unwrappedPage,
       breadcrumbs,
-      cmsPage,
-      category
```

Comment:
shouldn't also try to import it from the line:
`const {search, page, category, error} = useCms()`

Reply:
you're right, fixed!